crates/ln-dlc-node/src/node/mod.rs

```diff
@@ -116,6 +116,10 @@ pub struct Node<P> {
     pub(crate) alias: [u8; 32],
     #[cfg(test)]
     pub(crate) announcement_addresses: Vec<NetAddress>,
+
+    /// Liquidity-based routing fee in millionths of a routed amount. In
+    /// other words, 10000 is 1% and 5_000 is 0.5%
+    jit_funding_rate_millionth: Option<u32>,
```

Comment:
:wrench: Do we need this to be an `Option`? I think we can just make it mandatory.

Even if `new_app` doesn't care about it for now (because it doesn't make any sense since the app will not route payments anyway), it's not worth introducing an annoying `Option`. 

We just need to split the app and coordinator code at some point, which is the proper fix.